src/movies.js

```diff
@@ -1,16 +1,103 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+
+function getAllDirectors(movieArray) {
+    let directors = movieArray.map(function(item){
```

Comment:
instead of naming the parameter "item", you could name it "movie" or "eachMovie". It would be more understandable since every item is a movie object.